src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.IncrementalAnalyzerProcessor.cs

```diff
@@ -60,6 +64,13 @@ private partial class IncrementalAnalyzerProcessor
                     var lazyActiveFileAnalyzers = new Lazy<ImmutableArray<IIncrementalAnalyzer>>(() => GetIncrementalAnalyzers(_registration, analyzersGetter, onlyHighPriorityAnalyzer: true));
                     var lazyAllAnalyzers = new Lazy<ImmutableArray<IIncrementalAnalyzer>>(() => GetIncrementalAnalyzers(_registration, analyzersGetter, onlyHighPriorityAnalyzer: false));
 
+                    if (!initializeLazily)
+                    {
+                        // realize all analyzer right away
```

Comment:
when would someone not want to initialize lazily?

Reply:
comment is written in other places where initializeLazily is set. see that comment.